git-hub

```diff
@@ -759,9 +759,16 @@ class CloneCmd (object):
 	def run(cls, parser, args):
 		upstream = urlparse.urlsplit(args.repository).path
 		if upstream.endswith('.git'): upstream = upstream[:-4]
+		own_repo = True
+		proj = ''
```

Comment:
I just thought about how I could restructure this better. Pushing update in a minute.